pytest_fixtures/component/os.py

```diff
@@ -17,16 +20,28 @@ def default_os(
     """
     os = getattr(request, 'param', None)
     if os is None:
-        search_string = 'name="RedHat" AND (major="6" OR major="7" OR major="8")'
+        default_name = settings.supportability.content_hosts.default_os_name
```

Comment:
Have discussed this in the meeting today, raised by @ogajduse, here are some of thoughts coming from the meeting :

The way Satellite considers or creates`default_os`(The fixture name here as well) is when satellite installation is done and hence `Default OS` concept and this fixture should stick to just return the OS onto which Satellite is installed.

That being said, we don't need to search with criteria `major="6" OR major="7" OR major="8"` or in your latest patch `default_major_versions`. Hence searching just with `default_os_name` should be enough.

Also, the parametrization shouldn't be accepted for the default_os fixture as it's just meant to return the default. But, If the test demands for parametrized OS then we recommend creating a new fixture altogether.

@dosas Though the change we wanted to do here is not directly related to your PR we can simply accept it now and later someone from Satellite will update the fixture and create new fixture.

@dosas so Would you like to merge the PR or modify it as discussed here?

Reply:
The parametrization might be useful in the future but as far as I can see it is currently not used anywhere. So it can dropped/removed

So I can remove the parametrization and the remove the versions from the search string.

Reply:
Thanks @dosas.